spec.md

```diff
@@ -288,6 +288,27 @@ encapsulated within the `data` attribute.
 * Constraints:
   * OPTIONAL
 
+# Privacy & Security Considerations
+Interoperability is the primary driver behind this specification, enabling such
+behavior requires information to be made available *in the clear* resulting
+in the potential for information leakage.
+
+* Context Attributes
+
+  Sensitive information *SHOULD NOT* be carried or represented in context attributes.
```

Comment:
For consistency, let's remove the `*`s around the RFC2119 keywords

Reply:
Done